ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py

```diff
@@ -0,0 +1,78 @@
+"""
+This script is used to find out missed e2e profiles in upgrade config file.
+When we adding new upgrade config files, we can compare e2e config file with upgrade config file, then find out the jobs have not automated in upgrade.
+
+Here is the example to run this script:
+```
+e2e_yaml="/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml"
+upgrade_yaml="/home/jianl/1_code/release/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-upgrade-from-stable-4.16.yaml"
+
+python ci-operator/config/openshift/openshift-tests-private/tools/get_missed_jobs.py $e2e_yaml $upgrade_yaml
```

Comment:
Perhaps rename `get_missed_jobs.py` to `get_missed_upgrade_jobs.py` based on its purpose ?

Reply:
done